Gordon360/Controllers/ProfilesController.cs

```diff
@@ -121,6 +126,20 @@ public async Task<ActionResult<IEnumerable<AdvisorViewModel>>> GetAdvisorsAsync(
         return Ok(advisors);
     }
 
+    ///<summary>Get the privacy settings of a particular user</summary>
+    /// <returns>
+    /// All privacy settings of the given user.
+    /// </returns>
+    [HttpGet]
+    [Route("privacy_setting/{username}")]
```

Comment:
```suggestion
    [Route("{username}/privacy_setting")]
```
Since you're retrieving a property on the profile object for a user, this is the traditional way of ordering the URI.